### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by smburdick**
* Add Unitary Fund sponsorship tag
* Minor updates

### CODE REVIEW

**Comment by smburdick** [comment]
From the previous PR:

> > Installation guides are conventionally placed towards the top of `README.md`
> 
> Would it be possible to move the `Install all necessary packages` section of [`frontend/README.md`](https://github.com/QCHackers/tqec/blob/main/frontend/README.md) to the main README.md file? Or is there a reason to keep it separate?

I think we can keep those separate, since installing the frontend requires you to navigate to that directory, and it's only one command to install and another to compile and launch the app.

**Comment by nelimee** [comment]
> I think we can keep those separate, since installing the frontend requires you to navigate to that directory, and it's only one command to install and another to compile and launch the app.

Ok. At the very list I would nest the "installation" sections one level deeper under a `Installation` section.

**Comment by nelimee** [comment]
Miss-click, sorry for that.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # tqec
 
+[![Unitary Fund](https://img.shields.io/badge/Supported%20By-UNITARY%20FUND-brightgreen.svg?style=for-the-badge)](https://unitary.fund)
+
 Design automation software tools for Topological Quantum Error Correction
 
-## How to install (Python package)
+## How to install the backend (Python package)
```
Comment (nelimee) [inline README.md:7]:
```suggestion
## Installation
### Backend (Python package)
```

File: README.md [hunk 2]
```diff
@@ -50,11 +52,20 @@ python -m jupyter lab
 
 and to run some of the notebooks in the [`notebooks`](./notebooks/) directory.
 
+## Installing the frontend
```
Comment (nelimee) [inline README.md:55]:
```suggestion
### Frontend
```

**Review by nelimee — COMMENTED** [review]
I would merge both installation sections into their own `Installation` or `How to install` section.

**Review by nelimee — APPROVED** [review]

**Pull request merged**